Source/Core/VideoCommon/RenderBase.h

```diff
@@ -79,48 +79,52 @@ class Renderer
   virtual void RestoreState() {}
   virtual void ResetAPIState() {}
   virtual void RestoreAPIState() {}
+  // Some of the methods called by here assume g_renderer is initialized, therefore
```

Comment:
It ended up just being a single method call, which I've relocated elsewhere now, so it's not necessary any more.